docs/docs/10.1-animation.it-IT.md

```diff
@@ -44,7 +44,8 @@ var TodoList = React.createClass({
     return (
       <div>
         <button onClick={this.handleAdd}>Aggiungi Elemento</button>
-        <ReactCSSTransitionGroup transitionName="example" transitionEnterTimeout={500} transitionLeaveTimeout={300}>
+        <ReactCSSTransitionGroup transitionName="example" 
+        transitionEnterTimeout={500} transitionLeaveTimeout={300}>
```

Comment:
Let's format to match our JSX multi-line style:

```jsx
<ReactCSSTransitionGroup
  transitionName="example" 
  transitionEnterTimeout={500}
  transitionLeaveTimeout={300}>
```

Same goes for all of the other instances in this PR.